src/state-incentive-details.tsx

```diff
@@ -463,83 +484,85 @@ type Props = {
  * currently selected.
  */
 export const StateIncentives: FC<Props> = ({
-  firstResultsRef,
-  secondResultsRef,
+  resultsRef,
   response,
-  selectedProjects,
   emailSubmitter,
 }) => {
   const { msg } = useTranslated();
 
-  // We're filtering out IRA rebates in favor of state-specific handling.
-  const allEligible = response.incentives
-    // Forward compatibility for when only eligible incentives are returned from
-    // the API, with no eligible flag.
-    .filter(i => i.eligible === true || i.eligible === undefined)
-    .filter(i => !isIRARebate(i));
-
   // Map each project to all incentives that involve it. An incentive may
   // be in multiple projects, if it has multiple items and those items pertain
   // to different projects.
   const incentivesByProject = Object.fromEntries(
     Object.entries(PROJECTS).map(([project, projectInfo]) => [
       project,
-      allEligible.filter(incentive =>
+      response.incentives.filter(incentive =>
         incentive.items.some(item => projectInfo.items.includes(item)),
       ),
     ]),
   ) as Record<Project, Incentive[]>;
 
-  const projectsWithIncentives = (
-    Object.entries(incentivesByProject) as [Project, Incentive[]][]
-  )
-    .filter(([, incentives]) => incentives.length > 0)
-    .map(([project]) => project);
-
-  const interestedProjects = selectedProjects.sort((a, b) =>
-    shortLabel(a, msg).localeCompare(shortLabel(b, msg)),
-  );
+  const countOfProjects = Object.values(incentivesByProject).filter(
```

Comment:
<img width="774" alt="image" src="https://github.com/rewiringamerica/embed.rewiringamerica.org/assets/1214752/4a96c8da-938f-457d-9f48-29da488cdf49">
<img width="777" alt="image" src="https://github.com/rewiringamerica/embed.rewiringamerica.org/assets/1214752/959a4609-cae8-4230-af5d-b74e979a3ec1">

🤔 So, the counts don't seem to be right between the heading and the bullets + projects, am I missing something?

Reply:
Nah, I keep messing up the logic of including the IRA rebates in the counts :/